lib/v2/bilibili/cache.js

```diff
@@ -1,19 +1,19 @@
 const got = require('@/utils/got');
 const utils = require('./utils');
 const cheerio = require('cheerio');
+const config = require('@/config').value;
 
 module.exports = {
     getCookie: (ctx) => {
+        if (Object.keys(config.bilibili.cookies).length > 0) {
```

Comment:
I guess `getCookie` needs an update. Maybe filling `buvid3`, `b_nut`, `b_lsid`, `_uuid`, `buvid4` cookies when performing API requests will have less strict rate limit.

1. `buvid3` and `b_nut` will be provided by server when you visited the site first time, either `https://www.bilibili.com/` or `https://space.bilibili.com/1` will have them.

<details><summary>2. compute <code>b_lsid</code> and <code>_uuid</code></summary>

```js
// a
function randomHexStr(length) {
    let string = '';
    for (let r = 0; r < length; r++) {
        string += dec2HexUpper(16 * Math.random());
    }
    return padStringWithZeros(string, length);
}
// o
function dec2HexUpper(e) {
    return Math.ceil(e).toString(16).toUpperCase();
}
// s
function padStringWithZeros(string, length) {
    let padding = '';
    if (string.length < length) {
        for (let n = 0; n < length - string.length; n++) {
            padding += '0';
        }
    }
    return padding + string;
}
function lsid() {
    const e = Date.now().toString(16).toUpperCase();
    const lsid = randomHexStr(8) + '_' + e;
    return lsid;
}
function _uuid() {
    let e = randomHexStr(8);
    let t = randomHexStr(4);
    let r = randomHexStr(4);
    let n = randomHexStr(4);
    let o = randomHexStr(12);
    let i = Date.now();
    return e + '-' + t + '-' + r + '-' + n + '-' + o + padStringWithZeros((i % 100000).toString(), 5) + 'infoc';
}
```

</details>

3. GET `api.bilibili.com/x/frontend/finger/spi` with cookies `buvid3`, `b_nut`, `b_lsid`, `_uuid` which will return `buvid4` from `data.b_4`

4. continuing typical api request like `/x/space/wbi/acc/info` or `/x/space/wbi/arc/search`

Note: The value of the cookie `buvid_fp` and `fingerprint` are basically the output of [fingerprintjs](https://github.com/fingerprintjs/fingerprintjs)